src/renderer/src/lib/components/Login.svelte

```diff
@@ -42,16 +42,14 @@
       placeholder="***"
       type="password"
       tabindex="0"
-      bind:value={pass}
-    />
+      bind:value={pass} />
 
     <div class="mt-4 grid grid-cols-2 gap-2">
       <BackButton
         cancelText="Reset"
         on:cancel={() => {
           push('/locked/restore')
-        }}
-      />
+        }} />
```

Comment:
please check your linter + prettier settings to make sure that they are using the ones in the local scope